test/jdk/sun/security/x509/URICertStore/CRLReadTimeout.java

```diff
@@ -22,45 +22,101 @@
  */
 
 /*
- * @test
+ * @test id=0
  * @bug 8191808 8179502
  * @summary check that CRL download is interrupted if it takes too long
  * @modules java.base/sun.security.x509
  *          java.base/sun.security.util
  * @library /test/lib
- * @run main/othervm -Dcom.sun.security.crl.readtimeout=1
+ * @run main/othervm -Djava.security.debug=certpath -Dcom.sun.security.crl.readtimeout=1
  *      CRLReadTimeout 5000 false
- * @run main/othervm -Dcom.sun.security.crl.readtimeout=1s
+ */
+
+/*
+ * @test id=1
+ * @bug 8191808 8179502
+ * @summary check that CRL download is interrupted if it takes too long
+ * @modules java.base/sun.security.x509
+ *          java.base/sun.security.util
+ * @library /test/lib
+ * @run main/othervm -Djava.security.debug=certpath -Dcom.sun.security.crl.readtimeout=1s
```

Comment:
What's the difference between `-Dcom.sun.security.crl.readtimeout=1s` and `-Dcom.sun.security.crl.readtimeout=1`